docker-compose.rollkit.yml

```diff
@@ -135,6 +138,7 @@ services:
     environment:
       # TENDERMINT_RPC_URL should be the SimApp which is acting as a substitute
       # for Celestia (with IBC Eurekea enabled).
```

Comment:
This comment is no longer applicable to the line below it so should be moved or removed.
```suggestion
```